packages/clay-alert/src/ClayAlertBase.js

```diff
@@ -121,6 +109,26 @@ class ClayAlertBase extends Component {
 		}
 	}
 
+	/**
+	 * Sets the delayTime if passed, if it does not set the default, and starts.
+	 * @private
+	 */
+	_startTimer() {
+		if (
+			this.autoClose &&
+			(this.type === 'stripe' || this.type === 'toast')
+		) {
+			if (this.delayTime) {
+				this._delayTime = this.delayTime * 1000;
+			} else {
+				this._delayTime =
+					(this.element.querySelector('a') ? 10 : 5) * 1000;
```

Comment:
Remove this. Delay time should be no longer calculated depending on a link exists inside the alert. That should be a decision of the user.